fh.ws17.conterm.ContractFailureException
	at fh.ws17.conterm.Stock.getOtherStack(Unknown Source)
	at fh.ws17.conterm.StockControllerAbstractClass.umlade(Unknown Source)
	at fh.ws17.conterm.Terminal.entlade(Unknown Source)
	at fh.ws17.conterm.Terminal.abfertigung(Unknown Source)
	at fh.ws17.conterm.TerminalTestAufgabe1.testEinAusliefern(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)
Test von [fh.ws17.conterm]
Automatische Abnahme noch nicht bestanden @ 2017-12-04T20:50:15.198
Tests bestanden: 23
Tests fehlgeschlagen: 1

----------------------------------------------
testEinAusliefern(fh.ws17.conterm.TerminalTestAufgabe1)
java.lang.AssertionError




Status:200
